sdk/core/Azure.Core/src/ETag.cs

```diff
@@ -102,7 +102,7 @@ public string ToString(string format)
         {
             if (_value == null)
             {
-                return "<null>";
+                return string.Empty;
```

Comment:
This is sort of a breaking change. I wonder how much does the original issue hurts real customer scenarios.

Reply:
I don't imagine it would affect many real customer scenarios, but this behavior seems to be closer to what the RFC specifies. I'd guess both the breaking behavior and the current behavior would rarely cause a problem, so perhaps we should err on the side of conforming to the RFC.